src/akkudoktoreos/data/default.config.json

```diff
@@ -8,13 +8,13 @@
   "data_output_subpath": null,
   "elecprice_provider": null,
   "elecpriceimport_file_path": null,
-  "latitude": null,
```

Comment:
why did we set a default lat and long?

Reply:
So the system can be started and tested without any manual configuration.